content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -87,15 +87,15 @@ To run Windows Server Containers on Kubernetes, you'll need to set up both your
 ##### Linux Host Setup
 
 1. Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using.
-2. Configure Linux Master node using steps [here](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/creating-a-linux-master.md)
+2. Configure Linux control-plane machines using steps [here](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/creating-a-linux-master.md)
```

Comment:
`Configure Linux control-plane machines using steps [here]` -> `Configure a Linux control plane using the steps [here]`